index.template.html

```diff
@@ -4654,10 +4373,8 @@ <h6>Content Types</h6>
               additional parameters. In this specific case, the forms describe that reading this property with
               <code>http</code> or <code>coap</code> result in different content types. For structured media types, a
               Data Schema is generally provided in the affordance level as explained in
-              [[[#payload-bindings-dataschema]]] and
-              <a data-cite="WOT-THING-DESCRIPTION#sec-data-schema-vocabulary-definition">
-                in the Data Schema section of the TD specification</a
-              >. However, for unstructured data such as images and videos, a Data Schema is typically not available.
+              [[[#payload-bindings-dataschema]]] and [[#sec-data-schema-vocabulary-definition"]]. However, for
```

Comment:
[[#sec-data-schema-vocabulary-definition"]] does not link to anything. You can see it in the rendered version